src/components/FieldWrapper/FieldWrapper.js

```diff
@@ -26,7 +26,7 @@ export const FieldWrapper = React.forwardRef((props, ref) => {
                 <Button
                     isInline
                     context="link"
-                    onClick={handleClear}
+                    onMouseUp={handleClear}
```

Comment:
I would like to keep here `onClick`. We can pass to `FieldWrapper` prop `onClick` instead of `onMouseUp` and then Clear all should be work without open popup